target/linux/pistachio/config-4.4

```diff
@@ -157,6 +159,7 @@ CONFIG_DEVTMPFS=y
 CONFIG_DEVTMPFS_MOUNT=y
 CONFIG_DMADEVICES=y
 CONFIG_DMA_ENGINE=y
+CONFIG_DMA_MAYBE_COHERENT=y
```

Comment:
This option is not there in https://github.com/CreatorDev/linux/blob/openwrt-4.4.7/arch/mips/Kconfig
under MACH_PISTACHIO

only  DMA_NONCOHERENT is there